Gemfile

```diff
@@ -130,3 +130,4 @@ group :cucumber, :test do
   gem 'launchy'
   gem 'poltergeist'
 end
+gem 'rack-protection', '>= 2.0.1'
```

Comment:
This isn't useful. It looks to me from the Gemfile.lock that Whitehall has a transitive dependency on rack-protection, as Sidekiq uses it. I think it's only worth having direct dependencies in the Gemfile.